app/controllers/api/v4/invitations_controller.rb

```diff
@@ -6,11 +6,17 @@ class InvitationsController < ApplicationController
       include SetEvent
 
       skip_after_action :verify_authorized, only: [:index]
-      before_action :set_invitation, except: [:index]
+      before_action :set_invitation, except: [:index, :create]
```

Comment:
I think it might make a bit more sense to use an only here.
```suggestion
      before_action :set_invitation, only: [:show, :destroy]
```